sympy/assumptions/ask.py

```diff
@@ -1495,13 +1653,25 @@ def get_known_facts():
         Equivalent(Q.irrational | Q.rational, Q.real & Q.finite),
         Implies(Q.irrational, ~Q.rational),
         Implies(Q.zero, Q.even),
+        Equivalent(Q.extended_positive, Q.positive | Q.infinite),
+        Equivalent(Q.extended_negative, Q.negative | Q.infinite),
 
         Equivalent(Q.real, Q.negative | Q.zero | Q.positive),
+        Equivalent(Q.extended_real, Q.extended_negative | Q.zero | Q.extended_positive),
         Implies(Q.zero, ~Q.negative & ~Q.positive),
+        Implies(Q.zero, ~Q.extended_negative & ~Q.extended_positive),
```

Comment:
Since zero -> real -> finite -> !infinite and zero -> !negative & !positive I think (maybe...) this line is redundant.